### ISSUE TITLE
Completely broken on mobile

### ISSUE THREAD

**Issue by svivian** [phpmyadmin/phpmyadmin#14358]
### Steps to reproduce
1. Open a PMA instance on a mobile.
2. Type some text anywhere e.g. in the query editor (the bug happened to me in the "edit inline" feature and in the query console at the bottom of the page).

### Expected behaviour
The text to be typed.

### Actual behaviour
Seemingly completely random stuff happens - the page navigates away or random dialogs pop up. I think it could be some kind of shortcuts? I think it tried to go back when I pressed backspace. Pressing 'e' also tried to navigate away, and 's' brought up a settings dialog. Some other letters did things. But I have a text box focused so obviously it should be typing.

### Server configuration
**Operating system**: Ubuntu 16.04

**Web server:** nginx

**Database:** MySQL 5.7.22

**PHP version:** 7.0.30

**phpMyAdmin version:** 4.8.1 (from Github stable branch)

### Client configuration
**Browser:** Safari

**Operating system:** Apple iPhone 6, iOS 11.3.1

**Comment by ibennetch** [comment]
Confirmed with QA_4_8 and master, tested with iPhone/Safari on the SQL tab

**Comment by williamdes** [comment]
See #15224

**Comment by williamdes** [comment]
I assume this is fixed or solved 🤔

**Comment by svivian** [comment]
Yes it was a duplicate of the issue you linked, which is solved.
